### DISCUSSION TITLE
Roadmap? and Merge Pull Request?

### DISCUSSION THREAD

**Discussion by sevrick** [dmdorman/hero6e-foundryvtt#442] [General]
I was wondering if there was a road map for this. Either way is there more features planned, like compound powers? Also what is a Merge Pull request, Its I all I have noticed in the update? Thanks for the awesome work.

**Comment by phBalance** [comment]
PR = Pull Requests are a way of contributing to a project by sharing changes from your personal copy of the code. See [CONTRIBUTING.md](https://github.com/dmdorman/hero6e-foundryvtt/blob/main/CONTRIBUTING.md) for some links about pull requests.

I'm not aware of a formal roadmap but you can have a look at [FEATURES.md](https://github.com/dmdorman/hero6e-foundryvtt/blob/main/FEATURES.md) for things that need to be started, improved, and finished. There are active contributors so presumably things will continue to evolve.

Compound powers are partially supported in [#407](https://github.com/dmdorman/hero6e-foundryvtt/issues/407) but feel free to contribute even more!
